src/main/java/org/ssu/edu/teachua/db/entities/City.java

```diff
@@ -0,0 +1,30 @@
+package org.ssu.edu.teachua.db.entities;
+
+import lombok.Data;
+import org.ssu.edu.teachua.db.annotation.Column;
+import org.ssu.edu.teachua.db.annotation.TableDB;
+
+@TableDB(name = "cities")
+@Data
+public class City extends Entity {
+
+    @Column(name = "latitude")
+    private Double latitude;
+    @Column(name = "longitude")
+    private Double longitude;
+    @Column(name = "name")
+    private String name;
+
+    public Double getLatitude() {
```

Comment:
![image](https://user-images.githubusercontent.com/122312322/225591098-bceb1eba-b950-454a-9a9d-407501843bc0.png)
see Maksym's comments to my pull request.
You don't need to write method getLatitude()

Reply:
Thank you. Done.